guidelines/index.html

```diff
@@ -1433,30 +1434,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
  </section>
 
             
-            <section class="guideline" id="focus-appearance"><div class="header-wrapper"><h4 id="x2-4-13-focus-appearance"><bdi class="secno">Success Criterion 2.4.13 </bdi>Focus Appearance</h4><a class="self-link" href="#focus-appearance" aria-label="Permalink for Section 2.4.13"></a></div>
+            <section class="guideline" id="focus-appearance"><div class="header-wrapper"><h4 id="x2-4-13-focus-appearance"><bdi class="secno">達成基準 2.4.13 </bdi>フォーカスの外観</h4><a class="self-link" href="#focus-appearance" aria-label="Permalink for Section 2.4.13"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html">Understanding Focus Appearance</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#focus-appearance">How to Meet Focus Appearance</a></div><p class="conformance-level">(レベル AAA)</p>
 	<p class="change">[New]</p>
     
-    <p>When the keyboard <a href="#dfn-focus-indicator" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-focus-indicator-1" title="[New]">focus indicator</a> is visible, an area of the focus indicator meets all the following:</p>
+    <p>キーボードの<a href="#dfn-focus-indicator" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-focus-indicator-1" title="[New]">フォーカスインジケータ</a>が視覚的に表示される場合、フォーカスインジケータの領域が以下の全てを満たす。</p>
     <ul>
-        <li>is at least as large as the area of a 2 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-3" title="約 0.0213 度の視野角。">CSS pixel</a> thick <a href="#dfn-perimeter" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perimeter-1" title="[New]">perimeter</a> of the unfocused component or sub-component, and</li>
-        <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.</li>
+        <li>フォーカスされていないコンポーネント又はサブコンポーネントの幅 2 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-3" title="約 0.0213 度の視野角。">CSS ピクセル</a>の<a href="#dfn-perimeter" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perimeter-1" title="[New]">外周</a>の面積と少なくとも同じ大きさである。かつ、</li>
+        <li>同一ピクセルのフォーカスされている状態とフォーカスされていない状態との間で少なくとも 3:1 のコントラスト比がある。</li>
```

Comment:
これもわかりにくいですが原文もわかりにくいか。「同一ピクセルのフォーカスされている状態とフォーカスされていない状態」という言い回しだと「ピクセルがフォーカスされている」とも読めて、それがわかりにくい感じになっているのかもしれません。

たとえばこうしてみてはどうでしょうか?

```suggestion
        <li>フォーカスされている状態とフォーカスされていない状態において、同一ピクセルに 3:1 以上のコントラスト比がある。</li>
```

Reply:
なるほど、わかりやすくなりました！
一点、原文の「at least 3:1」は、他の達成基準 (1.4.3 コントラスト (最低限)、1.4.6 コントラスト (高度)、1.4.11 非テキストのコントラスト) において、いずれも「少なくとも n:n」という表記になっているので、以下のように修正する形で、いかがでしょうか？

> フォーカスされている状態とフォーカスされていない状態との間で、同一ピクセルに少なくとも 3:1 のコントラスト比がある。